test/runtests.jl

```diff
@@ -1,3 +1,25 @@
-using QuadraticModels
+# stdlib
+using SparseArrays, Test
 
-true
+# our packages
+using QuadraticModels, NLPModels, NLPModelsIpopt, LinearOperators
```

Comment:
Sorry for bothering with such small details, but please list them in alphabetical order.

Reply:
No problem ^^